### DISCUSSION TITLE
Welcome to Polkascan harvester Discussions!

### DISCUSSION THREAD

**Discussion by arjanz** [polkascan/harvester#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Georgezhang714** [comment]
I'm mr Muhammad from Nigeria africa I'm  here to get some knowledge of what i can fo with polkerscan

**Reply by arjanz** [reply]
This repository is a part of our open-source block explorer stack for Substrate based blockchains. You can find the composite repository for the application here: https://github.com/polkascan/explorer
